### PR TITLE
Main view

### PR DESCRIPTION
**Pull request by T40N**
Pull request for first view on my work with displaying the posts. Dont merge jet .

### CODE REVIEW

File: src/components/main/postGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import { createStyles, fade, Theme, makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+    kupka: {
```
Comment (arleta-jedrzejczak) [inline src/components/main/postGrid.tsx]: ???

**Review by arleta-jedrzejczak — APPROVED** [review]

File: src/components/main.tsx [hunk 2]
```diff
@@ -1,6 +1,109 @@
...
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    inputRoot: {
+      color: 'inherit',
+    },
+    inputInput: {
+      padding: theme.spacing(1, 1, 1, 0),
+      // vertical padding + font size from searchIcon
```
Comment (arleta-jedrzejczak) [inline src/components/main.tsx]: to delete

File: src/components/main.tsx [hunk 3]
```diff
@@ -1,6 +1,109 @@
...
+import { createStyles, fade, Theme, makeStyles } from '@material-ui/core/styles';
+import SearchIcon from '@material-ui/icons/Search';
+import axios from "axios";
+import { Input } from '@material-ui/core';
+import { TopBar } from './main/topBar';
+import {PostGrid} from './main/postGrid'
+
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
```
Comment (arleta-jedrzejczak) [inline src/components/main.tsx]: move styles to separate file

File: src/components/main/isLogedMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import { useHistory } from "react-router-dom";
+import AccountCircle from '@material-ui/icons/AccountCircle';
+
```
Comment (arleta-jedrzejczak) [inline src/components/main/isLogedMenu.tsx:6]: leave 1 row

File: src/components/main/isLogedMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+export const IsLogedMenu = () =>{
+
+    const history = useHistory();
+    
+    console.log(typeof(sessionStorage.getItem('id')));
```
Comment (arleta-jedrzejczak) [inline src/components/main/isLogedMenu.tsx]: to delete

Comment (arleta-jedrzejczak) [reply src/components/main/isLogedMenu.tsx]: use Prettier

File: src/components/main/postGrid.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory } from "react-router-dom";
+import { postsInterface } from './interfaces'
+
+
+const useStyles = makeStyles({
```
Comment (arleta-jedrzejczak) [inline src/components/main/postGrid.tsx]: move to separate file

File: src/components/user/user.tsx [hunk 7]
```diff
@@ -172,14 +172,16 @@ export const User = ({ id }) => {
...
-      .get(`https://damp-ridge-27698.herokuapp.com/users/${id}`)
+      .get(`https://damp-ridge-27698.herokuapp.com/users/606a0e06d2dade415814a66d`)
       .then((res) => {
         setUser(res.data);
         setPosts(res.data.posts);
       })
       .catch((err) => console.log(err));
   }, []);
 
+  console.log(posts);
```
Comment (arleta-jedrzejczak) [inline src/components/user/user.tsx:183]: to delete

File: src/components/user/user.tsx [hunk 8]
```diff
@@ -172,14 +172,16 @@ export const User = ({ id }) => {
 
   useEffect(() => {
     axios
-      .get(`https://damp-ridge-27698.herokuapp.com/users/${id}`)
+      .get(`https://damp-ridge-27698.herokuapp.com/users/606a0e06d2dade415814a66d`)
```
Comment (arleta-jedrzejczak) [inline src/components/user/user.tsx]: it should be hardcoded?

**Review by arleta-jedrzejczak — CHANGES_REQUESTED** [review]

File: src/components/main.tsx (hunk 3 above)
Comment (T40N) [reply src/components/main.tsx]: Done!

File: src/components/main/isLogedMenu.tsx (hunk 4 above)
Comment (T40N) [reply src/components/main/isLogedMenu.tsx:6]: Done!

File: src/components/main/isLogedMenu.tsx (hunk 5 above)
Comment (T40N) [reply src/components/main/isLogedMenu.tsx]: Done!

File: src/components/main/postGrid.tsx (hunk 6 above)
Comment (T40N) [reply src/components/main/postGrid.tsx]: Done!

File: src/components/user/user.tsx (hunk 8 above)
Comment (T40N) [reply src/components/user/user.tsx]: Done!

File: src/components/main.tsx (hunk 2 above)
Comment (T40N) [reply src/components/main.tsx]: Done!

**Review by arleta-jedrzejczak — APPROVED** [review]

**Pull request merged**